test/FunctionsV2/WebJobs.Extensions.DurableTask.Tests.V2.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net60</TargetFramework>
```

Comment:
What is the difference between these two?

Reply:
Sorry, this is an artifact of me going back and forth. They both seem to work, but the original `net6.0` seems to be what's in the official documentation. I'll revert this change.